docs/plugins/feature-flags.md

```diff
@@ -0,0 +1,64 @@
+---
+id: feature-flags
+title: Feature Flags
+description: Details the process of defining setting and reading a plugin feature flag.
+---
+
+Backstage offers the ability to define feature flags inside a plugin. This allows you to restrict parts of your plugin to those individual users who have toggled the feature flag to on.
+
+This page describes the process of defining setting and reading a plugin feature flag. If you are looking for using Feature flags with software templates that can be found under [Writing Templates](https://backstage.io/docs/features/software-templates/writing-templates#remove-sections-or-fields-based-on-feature-flags).
```

Comment:
This can be lower case just - so "feature flags" - to match the rest, sorry when I shared the text in the other PR I notice this after the fact

Reply:
No problem had not spotted it just copy and pasted it. Updated now.